src/test/java/seedu/eduke8/stats/StatsCalculatorTest.java

```diff
@@ -16,27 +16,28 @@
     private static final int HALF_PROGRESS_LEVEL_SCORE = 5;
     private static final int TOTAL_PROGRESS_LEVEL_SCORE = 10;
 
+    private StatsCalculator statsCalculator;
+
+    StatsCalculatorTest() {
+        statsCalculator = new StatsCalculator();
+    }
```

Comment:
Thanks for abstracting out the construction here so that it is not repeated elsewhere in the code!